src/org/aigents/nlp/lg/Loader.java

```diff
@@ -0,0 +1,88 @@
+package org.aigents.nlp.lg;
+
+import java.util.*;
+import java.io.*;
+
+public class Loader {
+	public static void main(String[] args) {
+		String data = "<dictionary-version-number>: V0v0v4+;\n" + 
```

Comment:
Ok for starter, but we will need data to be loaded by *.dict file, given the file name.